src/main/scala/scalafix/internal/sbt/BlockingCache.scala

```diff
@@ -0,0 +1,21 @@
+package scalafix.internal.sbt
+
+import scala.collection.mutable
+
+/** A basic thread-safe cache without any eviction. */
+class BlockingCache[K, V] {
```

Comment:
FWIW, if you don't mind using Java collections you can replace this class with
```scala
import java.{util => ju}
object BlockingCache {
  def empty[A, B] =
    ju.Collections.synchronizedMap(new ju.HashMap[A, B])
}

val cache = BlockingCache.empty[Int, String]
cache.computeIfAbsent(10, _ => "9")

cache // {10=9}
```
Those APIs are quite quirky however (return `AnyRef` in many places) so I like rolling a custom Scala solution instead.

Reply:
good idea - I kept the facade but swapped the implementation in e822f41